etc/schema.yaml

```diff
@@ -137,10 +137,11 @@ samples:
     useable: {                  type: 'string',  required: False, allowed: ['not marked', 'yes', 'no' ], default: 'not marked' }
     useable_comments: {         type: 'multilinestring',  required: False }
     useable_date: {             type: 'datetime', required: False }
+    useable_reviewer: {         type: 'string', required: False }
```

Comment:
Should we have this in run_elements as well?

Reply:
Yes you're right.
Added now